include/EncoderDecoder/Huffman/PaddedString.h

```diff
@@ -0,0 +1,12 @@
+#ifndef PADDED_STRING_H
+#define PADDED_STRING_H
+
+#include <string>
+
+class PaddedString {
+public:
+	uint8_t paddingSize;
```

Comment:
Put this data member after the other for less padding inside your struct

Reply:
TODO